scripts/tests_coverage.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# Butterfly build root
+BUTTERFLY_BUILD_ROOT=$1
+
+# Test Butterfly build root
+if [ ! -d $BUTTERFLY_BUILD_ROOT/api ]; then
```

Comment:
warning, someone putting butterfly's source root will pass this test as api folder also exist.